src/metabase/driver/postgres.clj

```diff
@@ -314,10 +314,15 @@
                         [:and [:!= :column_default nil] [:like :column_default [:inline "%nextval(%"]]]
                         [:!= :is_identity [:inline "NO"]]]]]
                 :database-required]
+               [:column_default :database-default]
                [[:or
                  [:and [:!= :column_default nil] [:like :column_default [:inline "%nextval(%"]]]
                  [:!= :is_identity [:inline "NO"]]]
-                :database-is-auto-increment]]
+                :database-is-auto-increment]
+               [[:= :is_nullable [:inline "YES"]]
+                :database-is-nullable]
+               [[:= :is_generated "ALWAYS"]
```

Comment:
I expected this to be `:!= "NEVER"` - are we sure which semantics we want?

Reply:
@wotbrew this is the thing I'd like to check with Katya

Reply:
Lets change it to `!= "NEVER"` and then change it again if there is a problem, unlikely to matter in short term.

Reply:
I just looked at the [postgres docs](https://www.postgresql.org/docs/current/infoschema-columns.html) again, expecting to find some nuance I had missed:

> is_generated character_data
> If the column is a generated column, then ALWAYS, else NEVER.

So not sure there is actually a problem here?